Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -68,14 +69,25 @@ public final class PledgeAmountViewModel: PledgeAmountViewModelType,
     )
 
     self.textFieldValue = stepperValue
-      .map { String(format: "%.0f", $0) }
-      .skipRepeats()
+      .map { value in
+        // Adds trailing zeros if the rounded number has non-zero remainder
+        // Removes trailing zeros and the decimal point otherwise
+        // Example:
+        //  25 => 25
+        //  25. => 25
+        //  25.0  => 25
+        //  25.00 => 25
+        //  25.1 => 25.10
+        //  25.10 = 25.10
+        let numberOfDecimalPlaces = value.truncatingRemainder(dividingBy: 1) == 0 ? 0 : 2
+        return String(format: "%.\(numberOfDecimalPlaces)f", value)
+      }
 
     self.currency = project
       .map { currencySymbol(forCountry: $0.country).trimmed() }
 
     self.stepperMinValue = minValue.mapConst(0)
-    self.stepperMaxValue = minValue.mapConst(Double.greatestFiniteMagnitude)
+    self.stepperMaxValue = minValue.mapConst(1_000_000_000)
```

Comment:
I'm not entirely sure what to do about this...we have to set `stepper.maxValue` since the default value is `100` but I'm not sure to what value. Since we allow inputs bellow pledge minimum, we should also allow inputs above pledge maximum in order to change the text color to red when the input is above that value and still allow stepper / text interactions.

Reply:
I think this is fine... it's an arbitrarily high number.